core/src/main/java/com/netflix/iceberg/hadoop/HadoopFileIO.java

```diff
@@ -0,0 +1,55 @@
+package com.netflix.iceberg.hadoop;
+
+import com.netflix.iceberg.FileIO;
+import com.netflix.iceberg.exceptions.RuntimeIOException;
+import com.netflix.iceberg.io.InputFile;
+import com.netflix.iceberg.io.OutputFile;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+
+public class HadoopFileIO implements FileIO {
+  private static final long serialVersionUID = 1L;
```

Comment:
No need for serialVersionUID because Iceberg makes no guarantees about Java serialization compatibility other than with the same Jar version everywhere.